crates/api_crud/src/post/update.rs

```diff
@@ -109,6 +110,12 @@ pub async fn update_post(
   )
   .await?;
 
+  let mut scheduled_publish_time = convert_published_time(data.scheduled_publish_time)?;
+  if orig_post.scheduled_publish_time.is_none() {
+    // cant schedule a post which is already published
+    scheduled_publish_time = None;
+  }
```

Comment:
No need for a mut, just `let scheduled_publish_time = if orig_post... { } else { convert..}.

Reply:
Changed, although I dont think mut is a problem.